src/burrow/burrow.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright 2017 HUAWEI. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * @file, definition of the Burrow class, which implements the Caliper's NBI for Hyperledger Burrow.
+ */
+
+'use strict';
+
+const fs = require('fs');
+const monax = require('@monax/burrow');
+const BlockchainInterface = require('../comm/blockchain-interface.js');
+const util = require('../comm/util.js');
+const logger = util.getLogger('burrow.js');
+const TxStatus = require('../comm/transaction');
+
+/* eslint-disable require-jsdoc */
```

Comment:
It would be better to provide the docs for the functions instead of disabling the jsdoc rule :)